docs/configuration/index.md

```diff
@@ -449,13 +449,10 @@ this message is used as an extension of the comment added by default by Packit.
 
 Use this key to be notified about errors of the downstream jobs (Koji build, Bodhi update, pull from upstream).
 The value can be a URL of any GitHub/GitLab/Pagure project including dist-git
```

Comment:
Sorry for nitpicking, but we should probably remove `including dist-git` from here as well.